tests/subdir.am

```diff
@@ -90,6 +90,7 @@ check_PROGRAMS += \
 endif
 
 tests/lib/cli/test_commands_defun.c: vtysh/vtysh_cmd.c
+	test -d tests/lib/cli || mkdir -p tests/lib/cli
```

Comment:
is this test needed while `mkdir -p` won't return as failed?

Reply:
I wasn't sure it consistently does that on all systems; is it guaranteed that `mkdir -p` doesn't fail if the directory already exists?

Reply:
```
% mkdir -p does/not/exists && echo ok
ok
[17:21] donatas-pc donatas /home/donatas
% mkdir -p does/not/exists && echo ok
ok
```
always returns true :)

Reply:
apparently POSIX requires this, so i'll drop the test.  Also need to look at NetBSD :(